### PR TITLE
Revert "Revert "Revert "Revert "renames broodmothers back to midwives""""

### PR DESCRIPTION
**Pull request by Bibby0110**
I was note made aware that a vote was made

Rockets count as 2 upvotes

### CODE REVIEW

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
Sorry Travis failed can't merge

**Comment by Bibby0110** [comment]
Looks fine to me

File: code/modules/mob/living/simple_animal/hostile/giant_spider.dm [hunk 1]
```diff
@@ -131,9 +131,9 @@
 	QDEL_NULL(set_directive)
 	return ..()
 
-//broodmothers are the queen of the spiders, can send messages to all them and web faster. That rare round where you get a queen spider and turn your 'for honor' players into 'r6siege' players will be a fun one.
+//midwives are the queen of the spiders, can send messages to all them and web faster. That rare round where you get a queen spider and turn your 'for honor' players into 'r6siege' players will be a fun one.
 /mob/living/simple_animal/hostile/poison/giant_spider/nurse/midwife
-	name = "broodmother"
+	name = "midwife"
```
Comment (1fbff5f83b23d39d38b1dfcb4cac8d9b) [inline code/modules/mob/living/simple_animal/hostile/giant_spider.dm]: Misspelled "broodmother"

**Review by 1fbff5f83b23d39d38b1dfcb4cac8d9b — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/hostile/giant_spider.dm (hunk 1 above)
Comment (Bibby0110) [reply code/modules/mob/living/simple_animal/hostile/giant_spider.dm]: intended
/mob/living/simple_animal/hostile/poison/giant_spider/nurse/midwife
**midwife**

**Comment by LoliconSlayer** [comment]
God fucking damn it

**Comment by MCHSL** [comment]
nope, failed

**Comment by Bibby0110** [comment]
image
Listen here you shit

**Review by alexkar598 — CHANGES_REQUESTED** [review]
doesnt compile

**Comment by Bibby0110** [comment]
Compiles

**Pull request closed without merging**